modules/nodes-cluster-limit-ranges-creating.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/limit-ranges.adoc
+
+[id="nodes-cluster-limit-creating_{context}"]
+= Creating a Limit Range
+
+To apply a limit range to a project, create a limit range
```

Comment:
Shouldn't this be a proper one step procedure, with associated markup?

Reply:
@jboxman I thought that the YAML should be here. But would that be redundant as I have the example above? And it is a large code block.